OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -274,6 +274,8 @@ void LoadAndDisplayNews(string newsURL, Widget newsBG)
 				{
 					if (!fetchedNews)
 					{
+						var newsURL = webServices.GameNews;
```

Comment:
This can move into a format parameter in the line below.

Reply:
It is modified in the process later one so it has to be cached.

Reply:
The next line appends a format string to this value. It causes confusion and adds technical debt to initialize the string with one value and then immediately append additional values using a format string.

Reply:
Moved it.

Reply:
I meant
```csharp
var newsURL = "{0}?version={1}&mod={2}&modversion={3}".F(
	webServices.GameNews
	Uri.EscapeUriString(Game.EngineVersion),
	Uri.EscapeUriString(Game.ModData.Manifest.Id),
	Uri.EscapeUriString(Game.ModData.Manifest.Metadata.Version));
```

Reply:
Changed it.